.github/actions/setvars/action.yaml

```
@@ -0,0 +1,13 @@
+name: 'Set environment variables'
+description: 'Configures environment variables for a workflow'
+inputs:
+  varFilePath:
+    description: 'File path to variable file or directory. Defaults to ./.github/variables/* if none specified and runs against each file in that directory.'
+    required: false
+    default: ./.github/variables/*
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        sed "" ${{ inputs.varFilePath }} >> $GITHUB_ENV
```

Is this because of the endline, that `sed` adds?

If not, consider switching to `cat`:
```suggestion
        cat ${{ inputs.varFilePath }} >> $GITHUB_ENV
```

---

I don't know, I copy-pasted it from an article.
I try to change it and we will see if it works

---

works